shell/app/modules/project/common/components/issue/edit-issue-drawer.tsx

```diff
@@ -740,7 +740,12 @@ export const EditIssueDrawer = (props: IProps) => {
           warnMessage.push(i18n.t('project:EstimateTime'));
         }
         if (params.issueManHour.elapsedTime === 0 && params.issueManHour.thisElapsedTime === 0) {
-          warnMessage.push(i18n.t('project:elapsedTime in time tracing'));
+          // filter out the working
+          const workingState = formData.issueButton.find(item => item.stateBelong === 'WORKING');
+          // When working exists and select working, don't warn
+          if (!workingState || (value.state !== workingState.stateID)) {
```

Comment:
Why not write as the comment: `if(workingState && (value.state === workingState.stateID))`